pom.xml

```diff
@@ -30,8 +30,6 @@
         <maven.release.plugin.version>2.5.3</maven.release.plugin.version>
         <maven.source.plugin.version>2.1.2</maven.source.plugin.version>
         <maven.toolchains.plugin.version>1.1</maven.toolchains.plugin.version>
-
-        <publish.package.name>org.opentestsystem.tds.common</publish.package.name>
```

Comment:
This is a Bintray artifact.  Now that we're going to be versioning our parent pom and submodules independently, they cannot share the same Bintray deployment "package"